### ISSUE TITLE
🐛 `Board` chevron missing dark mode style

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#10194]
## 🐛 Bug

`Board` chevron missing dark mode style.

## 🦋 Expected Behaviour

Chevron should have dark mode style that provides proper colour contrast.

## 📋 Steps to Reproduce

1. Run storybook
2. Navigate to Board component
3. Switch theme to GCDT Dark
4. Observe chevron icon adjacent to Collapsible info text

## 📸 Screenshot

<img width="1200" alt="Screen Shot 2024-04-25 at 16 52 53" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/2748f3bd-a148-4f57-a88c-eb711770fc53">

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] chevron should have dark mode style that matches the trigger icon used in `Accordion` component

**Comment by mnigh** [comment]
@GCTC-NTGC/designers any guidance on this one?

**Comment by substrae** [comment]
@mnigh 

Should just follow the `Accordion` component's styles. I'm assuming this isn't using that component directly, otherwise it would already be styled?

**Comment by mnigh** [comment]
> @mnigh
> 
> Should just follow the `Accordion` component's styles. I'm assuming this isn't using that component directly, otherwise it would already be styled?

Thanks @substrae, that should be enough for a dev to work with I think.

Correct, the `Board` component doesn't use the `Accordion` component: it uses the `Collapsible` component.
